helper_test.go

```diff
@@ -1177,6 +1201,47 @@ func createVariableSetVariable(t *testing.T, client *Client, vs *VariableSet, op
 	}
 }
 
+// Attempts to upgrade an organization to the business plan. Requires a user token with admin access.
```

Comment:
💅 Is this a code doc? You should prefix it "upgradeOrganizationSubscription attempts to upgrade..."